src/main/java/mate/academy/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy.dao.impl;
+
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+import mate.academy.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    private final SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+
+    @Override
+    public Order add(Order order) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = sessionFactory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(order);
+            transaction.commit();
+            return order;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can`t add order" + order, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+    }
+
+    @Override
+    public List<Order> getOrdersHistory(User user) {
+        try (Session session = sessionFactory.openSession()) {
+            String getOrdersByUserQuery = "FROM mate.academy.model.Order o JOIN FETCH o.tickets t "
```

Comment:
Why not use import instead of hardcoding it in the query?

Reply:
It's not a matter of using 'import'; I simply renamed my table from 'Order' to 'orders.' That's why I'm using the full name here.

Reply:
What I mean is that I've changed the column name from 'order' to 'orders' as described in the README.
![image](https://github.com/mate-academy/hibernate-order-hw/assets/124697095/ada14bb1-fc2a-45aa-940c-f60276999c3e)
![image](https://github.com/mate-academy/hibernate-order-hw/assets/124697095/f2bab3c9-4726-475b-975e-710c7ce00e6f)
So, now I need to either use 'orders' as the name or the full path here; otherwise, I'll encounter an error.
![image](https://github.com/mate-academy/hibernate-order-hw/assets/124697095/e1b23df7-4fea-4e5f-b9a0-b27c38917cb2)
![image](https://github.com/mate-academy/hibernate-order-hw/assets/124697095/19f0faa5-7842-4840-9bfa-008c5627740a)
That is why I used the full path qualifier, but it also works when I use name orders for the table:
![image](https://github.com/mate-academy/hibernate-order-hw/assets/124697095/2ffc82c0-de95-4fc4-9c49-59e4da0786dd)